Cargo.toml

```diff
@@ -43,14 +48,15 @@ tracing-futures = "0.2.2"
 tracing-subscriber = "0.2.1"
 url = "2.1.1"
 walkdir = "2.3.1"
-zstd = "0.5.1+zstd.1.4.4"
+zstd = "0.5.1"
```

Comment:
This removes a warning message that says the zstd version is already fully specified.